### PR TITLE
chore(actionbar): refactored the component to Functional components

### PR DESCRIPTION
**Pull request by anikethsaha**
**Type:**
Refactoring

The following has been addressed in the PR:

*  There is a related issue? https://github.com/verdaccio/ui/issues/116
*  Unit or Functional tests are included in the PR : NA

**Description:**
Migrated the `actionBar` component to functional component

### CODE REVIEW

**Comment by anikethsaha** [comment]
Should I update the Snapshots?
The fn component is working fine in my machine. All buttons are working as intended.

**Comment by bighuggies** [comment]
I also made a similar refactor before I saw that this PR had been opened: https://github.com/verdaccio/ui/compare/master...bighuggies:ah/action-bar-hooks

You should be able to use my changes to the tests as a reference to get CI working.

**Comment by anikethsaha** [comment]
> I also made a similar refactor before I saw that this PR had been opened: [master...bighuggies:ah/action-bar-hooks](https://github.com/verdaccio/ui/compare/master...bighuggies:ah/action-bar-hooks)
>
> You should be able to use my changes to the tests as a reference to get CI working.

Thanks man, I will refactor the tests soon.

File: src/components/ActionBar/ActionBar.tsx [hunk 1]
```diff
@@ -45,89 +46,77 @@ const ACTIONS = {
...
-            </Tooltip>
-          );
-        }
-      }
-      return component;
-    }, []);
+  if (renderList.length > 0) {
+    return (
+      <List>
+        <ActionListItem alignItems={'flex-start'} button={true}>
```
Comment (ayusharma) [inline src/components/ActionBar/ActionBar.tsx]: You can remove  `button={true}`, not needed here.

Comment (anikethsaha) [reply src/components/ActionBar/ActionBar.tsx]: Sure

**Comment by priscilawebdev** [comment]
@nikethsaha please do you mind if I take this over? 🙃

**Comment by anikethsaha** [comment]
> @nikethsaha please do you mind if I take this over? 🙃

Yea sure. Should I close this one ?

**Comment by priscilawebdev** [comment]
yes please :-)

**Pull request closed without merging**